src/test/java/picard/analysis/CollectInsertSizeMetricsTest.java

```diff
@@ -264,6 +264,24 @@ public void testMultipleOrientationsForHistogram() throws IOException {
         Assert.assertEquals(rResult, 0);
     }
 
+    @Test
+    public void testPercentCharInPdfFilename() throws IOException {
+        final File input = new File(TEST_DATA_DIR, "insert_size_metrics_test.sam");
+        final File outfile   = File.createTempFile("test_%_char", ".insert_size_metrics");
+        final File pdf = File.createTempFile("test_%_char", ".pdf");
+        pdf.deleteOnExit();
+
+        final String[] args = new String[] {
+            "INPUT="  + input.getAbsolutePath(),
+            "OUTPUT=" + outfile.getAbsolutePath(),
+            "Histogram_FILE=" + pdf.getAbsolutePath(),
+            "LEVEL=SAMPLE",
+            "LEVEL=LIBRARY",
+            "LEVEL=READ_GROUP"
+        };
+        Assert.assertEquals(runPicardCommandLine(args), 0);
```

Comment:
Would it make sense to check that the outfile is created with the expected name?  That is, still with a single %?  (just to future proof this).

Reply:
It would.  I'll add that in a new commit.

Reply:
Added below.